frontend/src/Pages/AdminManagement.tsx

```diff
@@ -167,8 +168,8 @@ export default function AdminManagement() {
                             enumType={{
                                 'Name (A-Z)': 'name_last asc',
                                 'Name (Z-A)': 'name_last desc',
-                                'Account Created ↓ ': 'created_at desc',
```

Comment:
I don't think this PR should've changed these icons. The v and the ^ character don't fit as well as the arrows. I'd recommend restoring it to the arrows and doing the same for `StudentManagement` as well.

![image](https://github.com/user-attachments/assets/41a10bc5-de0a-453c-bff8-5ae34da125bf)